src/Process/Deploy/CronProcessKill.php

```diff
@@ -74,7 +74,8 @@ private function killProcess($pid)
         try {
             $this->shell->execute("kill $pid");
         } catch (\RuntimeException $e) {
-            $this->logger->info('There is an error during killing the cron processes: ' . $e->getMessage());
+            $this->logger->info(sprintf('Couldn\'t kill process #%d it may be already finished', $pid));
```

Comment:
It can be possibly fixed by adding `$process->disableOutput()` to `Shell.php`

Reply:
We can re-declare `mustRun()` and throw our Exceptions without too verbose output

Reply:
It risky to re-declare `mustRun` method, because we should support many versions of Symfony console package.
We already catching Symfony exception in the `Process::execute` method so I changed the exception message there.

Reply:
It is happening only for one case, not really usable

Reply:
I changed the exception message for all shell exceptions. @see `Process::execute` method